modules/votes/ui/VoteQuorumStatus/VoteQuorumStatus.tsx

```diff
@@ -0,0 +1,88 @@
+import { Chip } from 'modules/shared/ui/Common/Chip'
+import { VoteStatusWrap } from './VoteQuorumStatusStyle'
+import { InfoIcon } from 'modules/shared/ui/Icons/info'
+import { VoteStatus } from '../../types'
+
+// import { VotePhasesTooltip } from '/modules/votes/ui/VotePhasesTooltip'
+
+interface Props {
+  totalSupply: number
+  minAcceptQuorum: number
+  yeaNum: number
+  nayNum: number
+  isEnded: boolean
+  status: VoteStatus
+}
+
+const isQuorumReached = ({
+  yeaNum,
+  nayNum,
+  totalSupply,
+  minAcceptQuorum,
+}: Props): boolean => {
+  if (totalSupply === 0) {
+    return false
+  }
+
+  const yeaQuorum = yeaNum / totalSupply
+  const nayQuorum = nayNum / totalSupply
+
+  return yeaQuorum > minAcceptQuorum || nayQuorum > minAcceptQuorum
+}
+
+const getWinningOption = (yeaNum: number, nayNum: number): 'Yes' | 'No' => {
+  return yeaNum > nayNum ? 'Yes' : 'No'
+}
+
+export function VoteQuorumStatus({
+  totalSupply,
+  minAcceptQuorum,
+  yeaNum,
+  nayNum,
+  isEnded,
+  status,
+}: Props) {
+  const quorumIsReached = isQuorumReached({
+    yeaNum,
+    nayNum,
+    totalSupply,
+    minAcceptQuorum,
+    isEnded,
+    status,
+  })
+
+  let winningOptionChip = null
+  if (quorumIsReached) {
+    const winningOption = getWinningOption(yeaNum, nayNum)
+    const winningVariant = winningOption === 'Yes' ? 'success' : 'danger'
+
+    winningOptionChip = (
+      <Chip variant={winningVariant}>Winning: {winningOption}</Chip>
+    )
+  }
+
+  let statusChip = null
```

Comment:
Maybe it's a nitpicking, but I don't like usage of `let` inside of components. I think there is a way to do the same without using `let`.